k2/csrc/utils.cu

```diff
@@ -1,59 +1,77 @@
 /**
  * @brief
- * utils
+ * context
  *
  * @copyright
- * Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey)
+ * Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey
+ *                                                   Meixu Song)
+ *                      Fangjun Kuang (csukuangfj@gmail.com)
  *
  * @copyright
  * See LICENSE for clarification regarding multiple authors
  */
 
-#include "k2/csrc/utils.h"
+#include "k2/csrc/utils.cuh"
+
+#include <cub/cub.cuh>
 
 namespace k2 {
 
-// See FillValues() where this is invoked.  It fills a region with
-// a constant value.
-__global__ void FillValuesKernel(int32_t *data, int32_t num_values,
-                                 int32_t value) {
-  int32_t job_idx = (blockIdx.x * blockDim.x + threadIdx.x),
-          num_jobs = (gridDim.x * blockDim.x);
-  for (; job_idx < num_values; job_idx += num_jobs) data[job_idx] = value;
+static int32_t RoundUpToNearestPowerOfTwo(int32_t n) {
+  K2_CHECK_GT(n, 0);
```

Comment:
This was a merge problem I think... I moved this code.

Reply:
Is it `RoundUpToNearestPowerOfTwo`? The master utils.cu still with it.

Reply:
ah I see.  should prob. be moved to math.{h,cc}.